docker-entrypoint.sh

```diff
@@ -1,157 +1,24 @@
-#!/usr/bin/env bash
-set -Eeo pipefail
-# TODO add "-u"
-
-# usage: file_env VAR [DEFAULT]
-#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
-# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
-#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
-file_env() {
-	local var="$1"
-	local fileVar="${var}_FILE"
-	local def="${2:-}"
-	if [ "${!var:-}" ] && [ "${!fileVar:-}" ]; then
-		echo >&2 "error: both $var and $fileVar are set (but are exclusive)"
-		exit 1
-	fi
-	local val="$def"
-	if [ "${!var:-}" ]; then
-		val="${!var}"
-	elif [ "${!fileVar:-}" ]; then
-		val="$(< "${!fileVar}")"
-	fi
-	export "$var"="$val"
-	unset "$fileVar"
-}
-
-isLikelyRedmine=
-case "$1" in
-	rails | rake | passenger ) isLikelyRedmine=1 ;;
-esac
-
-_fix_permissions() {
-	# https://www.redmine.org/projects/redmine/wiki/RedmineInstall#Step-8-File-system-permissions
-	if [ "$(id -u)" = '0' ]; then
-		find config files log public/plugin_assets \! -user redmine -exec chown redmine:redmine '{}' +
-	fi
-	# directories 755, files 644:
-	find config files log public/plugin_assets tmp -type d \! -perm 755 -exec chmod 755 '{}' + 2>/dev/null || :
-	find config files log public/plugin_assets tmp -type f \! -perm 644 -exec chmod 644 '{}' + 2>/dev/null || :
-}
-
-# allow the container to be started with `--user`
-if [ -n "$isLikelyRedmine" ] && [ "$(id -u)" = '0' ]; then
-	_fix_permissions
-	exec gosu redmine "$BASH_SOURCE" "$@"
+#!/bin/bash
+
+if [[ -v REDMINE_LANG ]];
+then
+echo "REDMINE_LANG is already set: ${REDMINE_LANG}"
+else
+echo "Set REDMINE_LANG variable to: en"
+export REDMINE_LANG="en"
 fi
```

Comment:
Personally, I prefer to have indent (tab? for bash script) between `then` ~ `else`  ~ `fi`.